traffic_portal/app/src/common/modules/form/user/edit/FormEditUserController.js

```diff
@@ -41,6 +49,10 @@ var FormEditUserController = function(user, $scope, $controller, $uibModal, $anc
         saveUser(user);
     };
 
+    $scope.resendRegistration = function(user, emailField) {
```

Comment:
do you know why it takes the `emailField` as an argument and does nothing with it?

Reply:
oops.. you are right.. I was on a different thought process.